scipy-1.0/subpackages.tex

```diff
@@ -10,9 +10,7 @@
 \begin{description}[leftmargin=!, labelwidth=\widthof{\bfseries \texttt{interpolate}}]
 \itemsep0em
 \item[\texttt{cluster}]
-    The \texttt{cluster} subpackage contains two modules:
-    \texttt{cluster.vq} provides vector quantization and $k$-means algorithms,
-    and \texttt{cluster.hierarchy} provides functions for hierarchical and
+    The \texttt{cluster} subpackage contains \texttt{cluster.vq}, which provides vector quantization and $k$-means algorithms, and \texttt{cluster.hierarchy}, which provides functions for hierarchical and
```

Comment:
These are indeed modules. Should I revert?

Reply:
Nah, what you did looks fine to me. Completely avoiding the confusion seems nice too.